types/jest-cli/index.d.ts

```diff
@@ -0,0 +1,92 @@
+declare module "jest-cli" {
```

Comment:
You don't need `declare module`,  just write the exports directly (as in `export type Argv = string[];` with no wrapper)

Reply:
@andy-ms is that for all things?

Reply:
Yes, export all the things